### DISCUSSION TITLE
How to update a dependency installed from a local wheel file?

### DISCUSSION THREAD

**Discussion by jingxue** [python-poetry/poetry#7321] [General]
In project B I add as a dependency a wheel file built from project A.  I make some changes in A, and build a new wheel file (but with the same version).  Now running `poetry update` in B doesn't seem to bring in the updates in the new wheel file.
The best workaround I can find is `poetry remove` the dependency and re-add the wheel file.  Is there a better way to do this?  Thanks.

**Comment by ralbertazzi** [comment]
If you made changes to A, then you should bump its version and release a different wheel.

Otherwise, remember that building a wheel is not a necessary step for using a local project in another local project. You can just use path dependencies https://python-poetry.org/docs/dependency-specification/#path-dependencies

**Comment by ralbertazzi** [comment]
If what I'm suggesting does not satisfy your use case, then please open an issue
